docs/DeveloperGuide.md

```diff
@@ -312,7 +327,7 @@ The mechanism allows the user to edit details of a fosterer in their profile pag
 
 Given below is an example usage scenario and how the mechanism behaves at each step, given that the user already opened person profile page:
 
-Step 1. The user enters the name of the field. e.g. "name". Since the normal person list is invisible, "name" is passed to `executeInView()` method in `MainWindow` class.
+Step 1. The user enters the name of the field. e.g. "name". Since the normal person list is invisible, "name" is passed to `executeInView()` method in `MainWindow` class (refer to the explanation in the section of [**Logic Component:AddressBookParser and ViewModeParser Classes**](#addressbookparser-and-viewmodeparser-classes)).
```

Comment:
might be better to standardise and call it fosterer list like you did at the start